### PR TITLE
Fix multiple failing tests

### PR DESCRIPTION
**Pull request by mefuller**
This PR fixes several failing tests (minor inconsistencies), and cleans up a few other things including correcting permissions on a faile and dropping the deprecated `nose`/`nosetests` in favor of `unittest` (which also provided the improved diagnostic information that led to fixing the most intransigent test)

### CODE REVIEW

File: arc/parserTest.py [hunk 1]
```diff
@@ -369,7 +369,7 @@ def test_parse_1d_scan_energies(self):
 
         path2 = os.path.join(ARC_PATH, 'arc', 'testing', 'rotor_scans', 'scan_1d_curvilinear_error.out')
         energies_2, angles_2 = parser.parse_1d_scan_energies(path=path2)
-        self.assertEqual(energies_2, None)
```
Comment (alongd) [inline arc/parserTest.py:372]: Minor: Please make the commit title shorter so it fits in one line, then add a commit description later as long as you'd like. Applies to other commits in this PR as well.

File: .gitignore [hunk 2]
```diff
@@ -29,6 +29,7 @@ examples/
 .coverage
 testing/*
 htmlcov/*
+arc/testing/
```
Comment (alongd) [inline .gitignore:32]: I think this should be removed - we add here files to use for testing. We do need git to track them when adding or modifying these files

File: Makefile [hunk 3]
```diff
@@ -7,7 +7,7 @@
 .PHONY : clean gcn gcn-cpu test test-unittests
 
 test test-unittests:
-	nosetests --nocapture --nologcapture --all-modules --verbose --with-coverage --cover-inclusive --cover-package=arc --cover-erase --cover-html --exe --cover-html-dir=testing/coverage
+	python3 -m unittest discover -p *Test.py -v
```
Comment (alongd) [inline Makefile:10]: Thanks! I'll check that it works smoothly for me as well

File: Makefile [hunk 4]
```diff
@@ -7,7 +7,7 @@
 .PHONY : clean gcn gcn-cpu test test-unittests
 
 test test-unittests:
```
Comment (alongd) [inline Makefile:9]: There's an empty commit in this PR, "remove executable permission from vectorsTest (make consistent)", please remove it

File: environment.yml [hunk 5]
```diff
@@ -25,7 +25,7 @@ dependencies:
   - mpmath
   - rmg::muq2
   - networkx
-  - nose
+  - nose
```
Comment (alongd) [inline environment.yml:28]: this commit just adds two white spaces, please remove the commit

File: arc/species/conformersTest.py [hunk 6]
```diff
@@ -586,7 +586,7 @@ def test_openbabel_force_field_on_rdkit_conformers(self):
                                      (2.51642, -0.38757, 0.25036))}]
         self.assertEqual(len(energies), 2)
         self.assertAlmostEqual(energies[0], 3.49581, 5)
```
Comment (alongd) [inline arc/species/conformersTest.py:588]: There's another empty commit in this PR, "fix permissions", please remove

**Review by alongd — COMMENTED** [review]
Thanks for these fixes! Please see some comments below
Please also see the [commit message style](https://github.com/ReactionMechanismGenerator/ARC/wiki/ARC-Contributor-Guidelines#commits-style) for this repo

File: Makefile (hunk 4 above)
Comment (mefuller) [reply Makefile:9]: It's not empty - the file permissions were incorrectly set to 755 when all other files are 644

File: arc/species/conformersTest.py (hunk 6 above)
Comment (mefuller) [reply arc/species/conformersTest.py:588]: This is erroneous and will be removed

File: docs/source/installation.rst [hunk 7]
```diff
@@ -55,7 +55,7 @@ Install dependencies
 
   Activate the ARC environment every time before you run ARC::
 
-     source activate arc_env
+    conda activate arc_env
```
Comment (alongd) [inline docs/source/installation.rst:58]: Let's merge this as PR #480

File: environment.yml (hunk 5 above)
Comment (mefuller) [reply environment.yml]: removed

File: .gitignore (hunk 2 above)
Comment (mefuller) [reply .gitignore]: There are files created in this directory during testing that should not be tracked - I'll revert now and later fixup the .gitignore file

**Review by alongd — APPROVED** [review]
Looking great, thanks!

**Pull request merged**